recipe/meta.yaml

```
@@ -14,6 +14,9 @@ source:
     - 0001-Fix-setup-for-windows.patch  # [win]
 
 build:
+  # This package does not support PyPy.
+  # See https://github.com/confluentinc/confluent-kafka-python/issues/1747#issuecomment-2112004625
+  skip: True  # [python_impl == 'pypy']
```

Please use lower case for YAML booleans:

```suggestion
  skip: true  # [python_impl == 'pypy']
```